include/deal_moon_conversion.hpp

```diff
@@ -0,0 +1,109 @@
+#include <deal.II/base/config.h>
+
+
+#ifdef DEAL_II_WITH_PARMOONOLITH
+
+#  include <deal.II/base/point.h>
+#  include <deal.II/base/quadrature.h>
+
+#  include <deal.II/fe/mapping.h>
+
+#  include <deal.II/grid/grid_generator.h>
+#  include <deal.II/grid/reference_cell.h>
+#  include <deal.II/grid/tria.h>
+
+#  include <moonolith_build_quadrature.hpp>
+#  include <moonolith_map_quadrature.hpp>
+#  include <moonolith_polygon.hpp>
+#  include <moonolith_vector.hpp>
+
+#  include <array>
+#  include <cassert>
+#  include <cmath>
+#  include <fstream>
+#  include <iostream>
+#  include <sstream>
+
+using namespace dealii;
+using namespace moonolith;
+
+
```

Comment:
can we enclose this in the moonolith namespace, i.e., `namespace moonolith { ... }`?

Reply:
Also, @FabCreda  has added the implementation of all the other functions. @FabCreda, can you move all function declarations from the test to the new file include file, and remove them from the test, while @fdrmrc implements the deal.II `compute_intersection` function?